docs/_blog/2025_q2_update.md

```diff
@@ -0,0 +1,55 @@
+
+There are also new projects and initiatives being built.
+- **Silent Payments.** The new [bdk-sp](https://github.com/bitcoindevkit/bdk-sp) crate brings BIP352 Silent Payments functionality to BDK. It has delivered a complete CLI implementation with wallet initialization, PSBT creation and signing, and blockchain scanning capabilities, and full compatibility with BIP352 test vectors and integration examples with bdk-tx.
+- **Transaction Building.** The experimental [bdk-tx](https://github.com/bitcoindevkit/bdk-tx) project represents an evolution of transaction building. This effort to decouple transaction building from the wallet, using rust-miniscript's planning module for optimal satisfaction-weight calculations and the new bdk_coin_select crate for policy-aware coin selection.
+- **Streaming Electrum Client.** The [electrum_streaming_client](https://github.com/bitcoindevkit/electrum_streaming_client) is a streaming, sans-IO Electrum client for asynchronous and blocking Rust applications. This crate provides low-level primitives and high-level clients for communicating with Electrum servers over JSON-RPC. It supports both asynchronous (futures/tokio) and blocking transport models.
+
+## Our Grantees in Action
+
+We're excited to welcome new Silver [corporate members](https://bitcoindevkit.org/blog/_2025q1-new-foundation-members/) the BDK Foundation and thank them for their financial support!
+- AnchorWatch
+- CleanSpark
+- Proton Foundation
+
+We're also excited to welcome new Associate members who are supporting the BDK Foundation by training & funding developers working on BDK!
+- Btrust
+- Vinteum
+
+- ** Grantees**
+BDK Project Maintainer
+- [Leonardo](https://github.com/oleonardolima), funded by Vinteum, continues his work on Tor integration for the Electrum and Esplora crates, enhancing privacy options for BDK users.
+
+Btrust Starter Grantees
+- [Itoro Ukpong](https://github.com/ItoroD), working on the bdk-ffi language bindings and Android example wallet.
+- [Peter Tyonum](https://github.com/tvpeter), upgrading bdk-cli to the latest bdk rust libraries and adding new features.
+
+Thunderbiscuit is taking a 2-month break from BDK using an HRF grant to create an iOS version of [Padawan Wallet](https://play.google.com/store/apps/details?id=com.coyotebitcoin.padawanwallet)
+
+- **BDK YouTube Channel.** We've launched our official [YouTube channel](https://www.youtube.com/@bitcoindevkit) featuring our Technical Talks series of 6 videos, covering topics:
+1. Language Bindings by [thunderbiscuit](https://github.com/thunderbiscuit)
+2. Silent Payments by [nymius](https://github.com/nymius)
+3. [Transaction Ordering](https://github.com/ValuedMammal/valuedmammal.github.io?tab=readme-ov-file#which-came-first) by [ValuedMammal](https://github.com/ValuedMammal)
+4. CLI by [Peter Tyonum](https://github.com/tvpeter)
+5. Compact Block Filters by [Robert Netzke](https://github.com/rustaceanrob)
+6. Mobile by [Matthew Ramsden](https://github.com/reez)
+
```

Comment:
You could add somewhere a line with the link to the whole playlist: https://www.youtube.com/playlist?list=PLFQTgyPgNM1iP9vqO6-Oic02x-MhxrNQu

Reply:
great idea, updated w that playlist link